osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerDisc.cs

```diff
@@ -126,7 +115,30 @@ protected override void Update()
                     .FadeTo(tracking_alpha, 250, Easing.OutQuint);
             }
 
-            this.RotateTo(currentRotation / 2, validAndTracking ? 500 : 1500, Easing.OutExpo);
+            this.RotateTo(currentRotation / 2, 500, Easing.OutExpo);
```

Comment:
What's the reason for this change?

Reply:
I can’t find any scenarios other than spun out that would make the spinner turn without being valid and tracking, so removing as a redundant condition.